tests/jenkins/TestDataPrepperReleaseArtifacts.groovy

```diff
@@ -25,7 +25,7 @@ class TestDataPrepperReleaseArtifacts extends BuildPipelineTest {
 
         String sourceImageRepository = 'http://public.ecr.aws/data-prepper-container-repository'
 
-        this.registerLibTester(new SignArtifactsLibTester( '.sig', 'linux',  'archive', null, null))
+        this.registerLibTester(new SignArtifactsLibTester( '.sig', 'linux', "${workspace}/archive", null, null))
```

Comment:
capital WORKSPACE here?

Reply:
This is part of the unit tests. This `workspace` is a variable defined here:

https://github.com/opensearch-project/opensearch-build/blob/ee396c460d6d6e4ecda08ff549d8f9655e0c9bc0/tests/jenkins/BuildPipelineTest.groovy#L22-L26